magento/magento2
# Moving admin export form to Ui components

### Description


Admin export form (`System > Export`) has been moved from blocks to Ui components.

The Pull Request adds the following Ui components:

1. `Magento_ImportExport::view/adminhtml/ui_component/export_form.xml`
2. `Magento_ImportExport::view/adminhtml/ui_component/export_filter_grid.xml`

### Manual testing scenarios

1. Go to the admin export page - `System > Export`
2. Make sure all elements of the Ui form are correctly loaded, there are no errors in the console
3. Choose `Entity Type = Products`, make sure grid with the product attributes is loaded without issues
4. Check the sort and filter features of the grid - all should be fine there
5. Export products through the button `Continue` below the grid, make sure the export was successful

### Notes


- All old php classes/layouts/phtml templates were marked as deprecated
- Broken mftf selectors were fixed to make tests green
- All new php classes were covered by unit tests

### Resolved issues:
1. [x] resolves magento/magento2#29587: Moving admin export form to Ui components
